src/FileServiceManager/FileService/Helpers/ErrorHandlingMiddleware.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace FileService.Helpers
+{
+    public class ErrorResponse
+    {
+        [JsonProperty("code", DefaultValueHandling = DefaultValueHandling.Ignore, Required = Required.Default, NullValueHandling = NullValueHandling.Ignore)]
+        public string Code { set; get; }
+        public string Error { set; get; }
+        [JsonProperty("stackTrace", DefaultValueHandling = DefaultValueHandling.Ignore, Required = Required.Default, NullValueHandling = NullValueHandling.Ignore)]
+        public string StackTrace { set; get; }
```

Comment:
If not using property then do not declare it, for stack trace I would advise populate it if the app is run in development mode

Reply:
++